src/main/java/MovieUI/Main.java

```diff
@@ -0,0 +1,85 @@
+package MovieUI;
+
+/**
+ * This class handles application wide tasks and services.
+ */
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.Pane;
+import javafx.stage.Stage;
+import object.MovieInfoObject;
+import ui.Ui;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public class Main extends Application {
+    private Stage mainWindow;
+    private Scene mainMoviesScene;
+    private BorderPane mainLayout;
+
+    public void start(Stage primaryStage) {
+        mainWindow = primaryStage;
+        setUp();
+
+        // Display window and the root scene
+        mainMoviesScene = new Scene(mainLayout);
+        mainWindow.setTitle("Entertainment Pro");
+        mainWindow.setScene(mainMoviesScene);
+        mainWindow.show();
+    }
+
+    /**
+     * This function se
+     * ts the scene for the window.
+     */
+    private void setUp() {
+        try {
+            FXMLLoader loader = new FXMLLoader();
+            loader.setLocation(getClass().getClassLoader().getResource("MainPage.fxml"));
+            mainLayout = loader.load();
+
+            // setup the controller's window and reference to this main application class
+            MovieHandler controller = loader.getController();
+            controller.setMainApplication(this);
+            controller.setWindow(mainWindow);
+        } catch (IOException e) {
+            Ui.printLine();
+            e.printStackTrace();
```

Comment:
Would the user (they are not developers) want to know the stacktrace of this exception?